p2p/lightchain.py

```diff
@@ -65,13 +71,18 @@ class LightPeerChain(PeerPoolSubscriber, BaseService):
     reply_timeout = REPLY_TIMEOUT
     headerdb: 'BaseAsyncHeaderDB' = None
 
-    def __init__(self, headerdb: 'BaseAsyncHeaderDB', peer_pool: PeerPool) -> None:
+    def __init__(
+            self,
+            headerdb: 'BaseAsyncHeaderDB',
+            peer_pool: PeerPool,
+            Chain: Type[BaseChain]) -> None:
         super().__init__()
         self.headerdb = headerdb
         self.peer_pool = peer_pool
         self._announcement_queue: asyncio.Queue[Tuple[LESPeer, les.HeadInfo]] = asyncio.Queue()
         self._last_processed_announcements: Dict[LESPeer, les.HeadInfo] = {}
         self._pending_replies: Dict[int, Callable[[protocol._DecodedMsgType], None]] = {}
+        self.Chain = Chain
```

Comment:
Why a capitalized variable name? Shouldn't it be `chain_class` to be consistent with other places?